catkin_ws/src/image_rotation/include/image_rotation/ImageAlgorithm.h

```diff
@@ -0,0 +1,17 @@
+#ifndef IMAGE_ALGORITHM_H
+#define IMAGE_ALGORITHM_H
+
+#include <opencv2/opencv.hpp>
+
+
+class ImageAlgorithm
```

Comment:
Consider replacing the class with a namespace to group utility functions. Refer: https://softwareengineering.stackexchange.com/questions/134540/are-utility-classes-with-nothing-but-static-members-an-anti-pattern-in-c , https://stackoverflow.com/questions/7345956/advantages-of-classes-with-only-static-methods-in-c

Reply:
we can look into this later.